src/quo2/components/drawers/drawer_buttons/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.drawers.drawer-buttons.view
+  (:require [react-native.core :as rn]
+            [quo2.components.icon :as icon]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.drawers.drawer-buttons.style :as style]))
+
+(defn render-bottom
+  [children]
+  [rn/view
+   {:flex-direction  :row
+    :justify-content :space-between}
+   [children]
+   [rn/view
+    {:style {:border-radius   40
+             :border-width    1
+             :margin-left     24
+             :height          28
+             :width           28
+             :justify-content :center
+             :align-items     :center
+             :border-color    (colors/alpha colors/white 0.05)}}
+    [icon/icon :arrow-right
+     {:color colors/white
+
+      :size  20}]]])
+
+(defn render-children-bottom
+  [children]
+  (if (or (string? children) (keyword children))
+    [render-bottom
+     (fn []
+       [text/text
+        {:size   :paragraph-2
+         :style  {:flex  1
+                  :color (colors/alpha colors/white 0.7)}
+         :weight :semi-bold}
+        children])]
+    [render-bottom children]))
+
+(defn render-children-top
+  [children]
+  (if (or (string? children) (keyword children))
+    [text/text
+     {:size   :paragraph-2
+      :style  {:color (colors/alpha colors/white 0.7)}
+      :weight :semi-bold}
+     children]
+    [children]))
+
+(defn card
+  [{:keys [on-press style heading children gap is-top?]}]
```

Comment:
Probably it's a personal preference, but I'd say that the boolean vars don't need the `is` prefix (such as Java conventions) because in Clojure it's enough if you add the `?` as suffix.

So I'd rename `is-top?` to `top?`.
Would be good to have feedback about this from someone else.

Reply:
Agree, I think there was a discussion of this topic before and it should be reflected in our guidelines, but didn't find it.

Reply:
It's not there @briansztamfater, although I find this a strong default for Clojure, I know it's not unanimous and it's not in the Clojure Style Guide (only predicates are mentioned https://guide.clojure.style/#naming-predicates).

So it's worth adding to the guidelines.

Reply:
I would appreciate if other people helped update the guidelines too, right now is a bit too concentrated on me. For the ones that already reached consensus and are quick (like this case of the question mark), they can even be done in the same PR if the contributor wants, whatever is easier to keep the docs up-to-date.

Reply:
I can update as part of this pr 👍